easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2.2-foss-2018b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'LocARNA'
+version = '1.9.2.2'
+
+homepage = 'http://www.bioinf.uni-freiburg.de/Software/%(name)s/'
+description = """LocARNA is a collection of alignment tools for the structural analysis of RNA.
+ Given a set of RNA sequences, LocARNA simultaneously aligns and predicts common structures for
+ your RNAs. In this way, LocARNA performs Sankoff-like alignment and is in particular suited for
+ analyzing sets of related RNAs without known common structure."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/s-will/%(name)s/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['76024e6d12a52e36a2d1aaffb09a7e7c1148ade05e7584530e21ce584c26e45b']
+
+dependencies = [
+    ('ViennaRNA', '2.4.11', '-Python-3.6.6'),
```

Comment:
@SethosII If `Python` is an (indirect) dependency, this should be reflected in the `versionsuffix`, can you open a follow-up PR to fix that for this easyconfig?

Reply:
@boegel I will do so. What about `RNAclust`? It would also have an indirect `Python` dependency afterwards but already also has `Perl`. Should I just keep the `versionsuffix` for it?

Reply:
It makes more sense to track the `Python` version being used in `versionsuffix`, since we stick to one single `Perl` version per toolchain generation anyway...

Reply:
@boegel #7708